amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -994,9 +996,9 @@ extern "C"
 	 * \returns A node reference <tt>\ref vx_node</tt>. Any possible errors preventing a
```

Comment:
Combine line #995 and #996 into one. They are both saying the same thing and will be better for doxygen.  This is true for all the fuctions.

Reply:
Yes, as mentioned in [this](https://github.com/LakshmiKumar23/MIVisionX/pull/32/files/897577f1145788898b0343d204be6e99728df33a#r1280922686) too

Reply:
Done